docs/src/pages/components/data-grid/overview/DataGridDemo.tsx

```diff
@@ -2,20 +2,21 @@ import * as React from 'react';
 import { DataGrid, ColDef } from '@material-ui/data-grid';
 
 const columns: ColDef[] = [
-  { field: 'id' },
-  { field: 'firstName' },
-  { field: 'lastName' },
+  { field: 'id', headerName: 'ID', width: 70 },
+  { field: 'firstName', headerName: 'First name', width: 130 },
+  { field: 'lastName', headerName: 'Last name', width: 130 },
   {
     field: 'age',
-    cellClassName: ['age'],
-    headerClassName: ['age'],
+    headerName: 'Age',
     type: 'number',
+    width: 90,
   },
   {
     field: 'fullName',
-    description: 'this column has a value getter and is not sortable',
-    headerClassName: 'highlight',
+    headerName: 'Full name',
+    description: 'This column has a value getter and is not sortable.',
```

Comment:
Make sure the headers are clean: https://master--material-ui-x.netlify.app/components/data-grid/#mit-version.